### DISCUSSION TITLE
Panic switches for Police, Ambulance, and Fire.

### DISCUSSION THREAD

**Discussion by akagarwal** [ufodone/envisalink_new#125] [Q&A]
Hi,
I am new to Home Assistant and recently migrated from HomeBridge + HomeKit. I got this integration to work perfectly except I don't see the three panic switches for Police, Ambulance, and Fire that were available in HomeBridge plugin. I am trying to setup an automation in HA to activate Fire panic automatically if no one is home and my smoke detectors activate. Can someone please help on how to setup these panic buttons. Thank you!

**Comment by Wildcat1** [comment]
Did you ever find a solution for this ?

**Comment by ufodone** [comment]
I may look at adding either service calls or additional switch entities to expose all three of these.  I just haven't had time to get to it.  Tracking that on issue #126.
